source/renderer/app/components/loading/syncing-connecting/SyncingConnectingStatus.tsx

```diff
@@ -219,10 +183,29 @@ export default class SyncingConnectingStatus extends Component<Props> {
       isNodeStopping,
       isNodeStopped,
       isTlsCertInvalid,
+      isVerifyingBlockchain,
       hasLoadedCurrentLocale,
       blockSync,
     } = this.props;
     if (!hasLoadedCurrentLocale) return null;
+
+    const {
+      connectingMessage,
+      connectingDescription,
+    } = this._getConnectingMessage();
+
+    const isConnectingMessage =
+      connectingMessage === messages.connecting ||
+      connectingMessage === messages.reconnecting;
+
+    if (isVerifyingBlockchain && isConnectingMessage) {
```

Comment:
Can we check if `this.props.cardanoNodeState` is `CardanoNodeStates.RUNNING` instead of looking at `connectingMessage`?

Reply:
Also, just a question, is it correct to check `isVerifyingBlockchain`? It looks similar to the second step:
![image](https://user-images.githubusercontent.com/6684554/154331681-451dac6d-41b6-4a58-93e8-ff3888cc0239.png)

Reply:
> Can we check if this.props.cardanoNodeState is CardanoNodeStates.RUNNING instead of looking at connectingMessage?

Agree, though I don't wanna mess with the logic that was here already. Those two messages are set for the `RUNNING` state but also for any other as they are in the `default` option of a `switch`. I will check other states and decide whether your assumption is valid.

> Also, just a question, is it correct to check isVerifyingBlockchain? It looks similar to the second step

TBH I don't know what the `isVerifyingBlockchain` flag stands for. I just used the logic that was already here. If there is a time I can investigate that.

Reply:
Regarding checking `CardanoNodeStates.RUNNING` instead of specific messages I decided not to do that. In order to  do that properly I would need to mess up with the logic of other states in order to eliminate the `default` case of the `switch`. I don't wont to do that in this PR.